yourgov/pages/app/staff/invite-staff-member/index.tsx

```diff
@@ -139,7 +141,7 @@ const Page: NextPageWithLayout = () => {
 							/>
 
 							<TextInput
-								autoComplete="mobile"
+								autoComplete="tel"
```

Comment:
Is this an arc thing? https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#mobile